nixos/doc/manual/release-notes/rl-2103.xml

```diff
@@ -250,6 +250,18 @@
   <title>Other Notable Changes</title>
 
   <itemizedlist>
+   <listitem>
+    <para>
+     The Mailman NixOS module (<literal>services.mailman</literal>)
```

Comment:
Use `<option>` for option names, that way they should get linked AFAIU

Reply:
oh well, I'm running a bisect now, I should make separate worktrees for when I bisect :rofl:

don't worry I'll make a new worktree in RAM real quick